www/assets/js/editor.js

```diff
@@ -10,3 +10,12 @@ function charCounter(text, counter) {
 	});
 	$(text).trigger('keyup');
 }
+
+function sendMail(){
+    var mail = document.getElementById('mailbtn').value
+    var title = document.getElementById('title').value
+    var author = document.getElementById('user').value
+    document.getElementById('context').value = 'Foto(s) door ' + author + ' gemaild\n' + document.getElementById('context').value;
```

Comment:
Check even of de context deze exacte tekst al bevat, voor het geval de gebruiker meerdere keren op de knop drukt.